openssl-sys/src/kdf.rs

```diff
@@ -0,0 +1,220 @@
+use libc::*;
+use *;
+
+const EVP_PKEY_CTRL_TLS_MD: c_int = EVP_PKEY_ALG_CTRL;
+const EVP_PKEY_CTRL_TLS_SECRET: c_int = EVP_PKEY_ALG_CTRL + 1;
+const EVP_PKEY_CTRL_TLS_SEED: c_int = EVP_PKEY_ALG_CTRL + 2;
+const EVP_PKEY_CTRL_HKDF_MD: c_int = EVP_PKEY_ALG_CTRL + 3;
+const EVP_PKEY_CTRL_HKDF_SALT: c_int = EVP_PKEY_ALG_CTRL + 4;
+const EVP_PKEY_CTRL_HKDF_KEY: c_int = EVP_PKEY_ALG_CTRL + 5;
+const EVP_PKEY_CTRL_HKDF_INFO: c_int = EVP_PKEY_ALG_CTRL + 6;
+const EVP_PKEY_CTRL_HKDF_MODE: c_int = EVP_PKEY_ALG_CTRL + 7;
+const EVP_PKEY_CTRL_PASS: c_int = EVP_PKEY_ALG_CTRL + 8;
+const EVP_PKEY_CTRL_SCRYPT_SALT: c_int = EVP_PKEY_ALG_CTRL + 9;
+const EVP_PKEY_CTRL_SCRYPT_N: c_int = EVP_PKEY_ALG_CTRL + 10;
+const EVP_PKEY_CTRL_SCRYPT_R: c_int = EVP_PKEY_ALG_CTRL + 11;
+const EVP_PKEY_CTRL_SCRYPT_P: c_int = EVP_PKEY_ALG_CTRL + 12;
+const EVP_PKEY_CTRL_SCRYPT_MAXMEM_BYTES: c_int = EVP_PKEY_ALG_CTRL + 13;
+
+const EVP_PKEY_HKDEF_MODE_EXTRACT_AND_EXPAND: c_int = 0;
+const EVP_PKEY_HKDEF_MODE_EXTRACT_ONLY: c_int = 1;
+const EVP_PKEY_HKDEF_MODE_EXPAND_ONLY: c_int = 2;
+
+pub unsafe extern "C" fn EVP_PKEY_CTX_set_tls1_prf_md(
```

Comment:
Functions defined from macros elsewhere in the crate use the standard Rust ABI rather than the C ABI. Probably doesn't matter too much, but it seems best to match that here.